### PR TITLE
Edits to text in Get API doc

### PR DESCRIPTION
**Pull request by dmeiss**
Minor edits to text for punctuation and word usage

### CODE REVIEW

File: docs/reference/docs/get.asciidoc [hunk 1]
```diff
@@ -238,7 +238,7 @@ You can also use the same source filtering parameters to control which parts of
 
 [source,js]
 --------------------------------------------------
-GET twitter/_doc/1/_source?_source_includes=*.id&_source_excludes=entities'
```
Comment (dmeiss) [inline docs/reference/docs/get.asciidoc]: OK to delete this stray single quote?

Comment (danielmitterdorfer) [reply docs/reference/docs/get.asciidoc]: Yes, that looks ok to me.

**Comment by danielmitterdorfer** [comment]
@elasticmachine test this please

**Review by danielmitterdorfer — APPROVED** [review]
Thanks for these fixes @dmeiss! Looks good to me.

**Pull request merged**